modules/node_modules/@frogpond/ccc-lib/html.js

```diff
@@ -0,0 +1,25 @@
+import {toLaxTitleCase} from '@frogpond/titlecase'
+
+import _jsdom from 'jsdom'
+const {JSDOM} = _jsdom
+
+export {encode, decode} from 'html-entities'
+
+// Html
+
+export function parseHtml(string) {
+	return JSDOM.fragment(string).textContent.trim()
+}
+
+export function innerTextWithSpaces(elem) {
+	return JSDOM.fragment(elem).split(/\s+/u).join(' ').trim()
+}
+
+export function removeHtmlWithRegex(str) {
+	return str.replace(/<[^>]*>/gu, ' ')
+}
+
+export function fastGetTrimmedText(str) {
```

Comment:
Since we have JSDOM here and aren't resource constrained, I'd like to remove this fn in favor of a JSDOM-based solution

Reply:
Thanks for mentioning this. We will go with the `textContent` solution.